emcc.py

```diff
@@ -769,6 +769,10 @@ def array_contains_any_of(hay, needles):
       if n in hay:
         return True
 
+  # relaxed-simd implies simd128.
+  if '-mrelaxed-simd' in user_args:
+      user_args += ['-msimd128']
+
```

Comment:
```suggestion


```
flake8 again

Reply:
Think this is the wrong file, the 2 line spacing should be in the new decorator in test_core.py